tests/sentry/utils/json/tests.py

```diff
@@ -33,12 +33,12 @@ def test_escape(self):
         res = "<script>alert('&');</script>"
         assert json.dumps(res) == "\"<script>alert('&');</script>\""
         assert (
-            json.dumps(res, escape=True)
-            == '"\\u003cscript\\u003ealert(\\u0027\u0026\\u0027);\\u003c/script\\u003e"'
+            json.dumps(res, escape=True).encode("utf-8")
```

Comment:
Wondering if it makes sense to use `text_type` instead, and compare strings? I guess it's equivalent either way